lib/index.js

```diff
@@ -41,7 +41,7 @@ function createPreprocessor (args, config, logger, helper)
                 })
                 .catch(function (error)
                 {
-                    log.error('%s\n  at %s:%d', e.message, file.originalPath, e.location.first_line)
+                    log.error('Failed to process "%s".\n  %s', file.originalPath, error.message);
```

Comment:
I did not want to change too much, but it seems in case of syntax errors, the error contains a property called `codeFrame` with context for the syntax error. This could be included in the output to yield nicer error reports:
```JavaScript
if (error.hasOwnProperty("codeFrame")) {
    log.error('Failed to process "%s":\n%s\n%s', file.originalPath, error.message, error.codeFrame);
}
else {
    log.error('Failed to process "%s":\n%s', file.originalPath, error.message);
}
```
**Output from syntax error:**
```
21 02 2016 18:09:06.932:ERROR [preprocessor.rollup]: Failed to process "/Users/Mats/Documents/Code/redux-preact-test/test/test-reducers.js".
  /Users/Mats/Documents/Code/redux-preact-test/src/actions.js: Unexpected token (3:7)
  1 | import {ensure, isArrayOf, isInstanceOf, isNonEmptyArrayOf, isNonEmptyString, isId} from "./validation";
  2 | import generatePushId from "./3rdparty/generate-pushid";
> 3 | impart Vector from "./vector";
    |        ^
  4 | 
  5 | const isArrayOfIds = isArrayOf(isId);
  6 | const isVector = isInstanceOf(Vector);
```